src/makeRobot.js

```diff
@@ -38,6 +38,57 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
```

Comment:
```suggestion
    name,
    version,
    wheels,
```
можна використовувати скорочений запис